### PR TITLE
Add slowmode flood error

### PR DESCRIPTION
**Pull request by Andrebcd4**

### CODE REVIEW

**Comment by Lonami** [comment]
The changes are good, but I think we should special-case `SLOWMODE` to become `SlowMode` and not `Slowmode`. This is done here:
https://github.com/LonamiWebs/Telethon/blob/4f6e5c5f5a7796459a02ad51f6bae90ea2c11d41/telethon_generator/parsers/errors.py#L31

File: telethon_generator/data/errors.csv [hunk 1]
```diff
@@ -228,6 +228,7 @@ SESSION_PASSWORD_NEEDED,401,Two-steps verification is enabled and a password is
 SESSION_REVOKED,401,"The authorization has been invalidated, because of the user terminating all sessions"
 SHA256_HASH_INVALID,400,The provided SHA256 hash is invalid
 SHORTNAME_OCCUPY_FAILED,400,An error occurred when trying to register the short-name used for the sticker pack. Try a different name
+SLOWMODE_WAIT_X,420,A wait of {seconds} seconds is required
```
Comment (Lonami) [inline telethon_generator/data/errors.csv]:
```suggestion
SLOWMODE_WAIT_X,420,A wait of {seconds} seconds is required before sending another message in this chat
```

File: telethon/client/telegrambaseclient.py [hunk 2]
```diff
@@ -118,7 +118,7 @@ class TelegramBaseClient(abc.ABC):
 
         flood_sleep_threshold (`int` | `float`, optional):
             The threshold below which the library should automatically
-            sleep on flood wait errors (inclusive). For instance, if a
+            sleep on flood wait and slowmode wait errors (inclusive). For instance, if a
```
Comment (Lonami) [inline telethon/client/telegrambaseclient.py]:
```suggestion
            sleep on flood wait and slow mode wait errors (inclusive). For instance, if a
```

File: telethon_generator/parsers/errors.py [hunk 3]
```diff
@@ -28,7 +28,8 @@ def _get_class_name(error_code):
         )
 
     return snake_to_camel_case(
-        error_code.replace('FIRSTNAME', 'FIRST_NAME').lower(), suffix='Error')
+        error_code.replace('FIRSTNAME', 'FIRST_NAME')\
+        .replace('SLOWMODE', 'SLOW_MODE').lower(), suffix='Error')
```
Comment (Lonami) [inline telethon_generator/parsers/errors.py]:
```suggestion
                  .replace('SLOWMODE', 'SLOW_MODE').lower(), suffix='Error')
```

**Comment by Andrebcd4** [comment]
By the way, error description was taken from FLOOD_WAIT_X. Maybe you want to update it.

**Pull request merged**